advancedFunctions/detonatorTimer.js

```diff
@@ -0,0 +1,34 @@
+//setTimout recursion
+const detonatorTimerRecr = (delay) => {
+    if (delay < 0) {
+      return
+    }
+    console.log(delay === 0 ? 'BOOM!' : delay);
+    delay -= 1;
+    setTimeout(detonatorTimerRecr, 1000, delay);
```

Comment:
do not use the third param in the setTimeout

Reply:
Done